libbeat/tests/system/idxmgmt.py

```diff
@@ -0,0 +1,88 @@
+from elasticsearch import NotFoundError
+from nose.tools import raises
+import datetime
+
+
+class IdxMgmt(object):
+
+    def __init__(self, client):
+        self._client = client
+
+    def clean(self, prefix):
+
+        # Delete existing indices and aliases
+        try:
+            self._client.transport.perform_request('DELETE', "/" + prefix + "*")
+        except:
+            pass
```

Comment:
shall we log a message for debugging purposes?

Reply:
Changed to only catch `NotFoundError`.